### ISSUE TITLE
There is 6 order's status in Thelia 2.2.x

### ISSUE THREAD

**Issue by InformatiqueProg** [thelia-modules/ImportT1#15]
In ordersImport
// Delete custom order status
OrderStatusQuery::create()->filterById(5, Criteria::GREATER_THAN)->delete();

**Comment by roadster31** [comment]
I don't understand the issue. What's the problem ?

**Comment by InformatiqueProg** [comment]
If i'm right, this mean that the orderstatus greater than 5 will be deleted, no ?
Now we have 6,
INSERT INTO `order_status`(`id`, `code`, `created_at`, `updated_at`) VALUES
(1, 'not_paid', NOW(), NOW()),
(2, 'paid', NOW(), NOW()),
(3, 'processing', NOW(), NOW()),
(4, 'sent', NOW(), NOW()),
(5, 'canceled', NOW(), NOW()),
(6, 'refunded', NOW(), NOW());
The ```refunded``` will be deleted...

**Comment by roadster31** [comment]
Thelia 1 have five. The sixth could be anything.

**Comment by roadster31** [comment]
True. But as in Thelia 1 we have only 5 statuses, the 6th one is necessarily a custom status, which could mean anything.

During the import process, we have to ignore theses T1 custom statuses, as we have to ignore T2 custom statuses.

**Comment by InformatiqueProg** [comment]
Ok, now i understand why you deleted it...

what about add T1 order statuses > 5 in T2

**Comment by roadster31** [comment]
Good idea !
